package.json

```diff
@@ -27,6 +27,8 @@
     "schema-drift": "babel-node --no-warnings --extensions '.ts,.js' ./scripts/schema-drift.ts",
     "start": "yarn run dev",
     "test": "yarn type-check && yarn lint && jest",
+    "test:jest:v1": "yarn jest --config jest.config.v1.js --maxWorkers 3",
+    "test:jest:v2": "yarn jest --config jest.config.v2.js --maxWorkers 3",
```

Comment:
We should be careful with the extra workers here. Especially if you're parallelizing the execution of the tests. 

From [Circle's docs](https://circleci.com/docs/2.0/collect-test-data/#jest):
>Note: When running Jest tests, please use the --runInBand flag. Without this flag, Jest will try to allocate the CPU resources of the entire virtual machine in which your job is running. Using --runInBand will force Jest to use only the virtualized build environment within the virtual machine.

>For more details on --runInBand, refer to the Jest CLI documentation. For more information on these issues, see [Issue 1524](https://github.com/facebook/jest/issues/1524#issuecomment-262366820) and [Issue 5239](https://github.com/facebook/jest/issues/5239#issuecomment-355867359) of the official Jest repository

The challenge here is that are tests could fail in non-deterministic ways, depending on what other resources are running on the CI runner instance.

I'm good w/ trying it, but we'll need to observe and potentially roll back if we start seeing a lot of failures.

Reply:
For sure 👍 This worker count of 3 is what we landed on in Force and its been going pretty good so far. Will keep an eye on it.